FlightTelemetry/FlightTelemetry.ino

```diff
@@ -6,15 +6,19 @@
 #include <Adafruit_BNO055.h>
 #include <avr/pgmspace.h>
 
-#define TELEMETRY_SERIAL Serial1
+#define TELEMETRY_SERIAL Serial2  //Serial fine for Arduino Boards + Due
+//Teensy 3.6 has to use Serial1 or higher
 #include <Telemetry.h>
 
 Adafruit_BNO055 bno = Adafruit_BNO055(55);
 RTC_DS1307 rtc;
 File dataFile;
 char filename[] = "DATA000.csv";
 
-#define LED           10
+#define LED     BUILTIN_SDCARD  //use BUILTIN_SDCARD for Teensy 3.6
+//also change line ~46 to: 
+//if (!SD.begin(BUILTIN_SDCARD)) { TELEMETRY_SERIAL.println(F("SD err")); }
```

Comment:
Unclear comment - this change seems to have already been made?

Reply:
is the LED pin only used to blink when there is an error? I think I previously put BUILTIN_SDCARD there but I don't think that makes any sense. I just wanted to see if you concur

Reply:
I think so?  I didn't write this code.  The param passed to `SD.begin()` is the chip select, which should definitely not be the same as the LED.